### ISSUE TITLE
Reconciled transactions colors make it very difficult to read

### ISSUE THREAD

**Issue by feerlessleadr** [Boren/dark-ynab#2]
HI - thanks for this, everything looks great. 

I noticed today when searching for a transaction that the colors on reconciled transactions make the text unreadable with the theme enabled. See below screenshot.

![image](https://user-images.githubusercontent.com/288895/79815095-cb9ce900-834d-11ea-9cd5-bad27a51f403.png)

Thanks

**Comment by Boren** [comment]
![image](https://user-images.githubusercontent.com/1568580/80124685-e6ab6b00-8590-11ea-813a-c89b118b54b8.png)

In what browser and OS do you see this? I'm unable to reproduce here.

**Comment by feerlessleadr** [comment]
This is on Brave browser (Version 1.8.78 Chromium: 81.0.4044.92 (Official Build) beta (64-bit)) on Windows 10.

**Comment by Boren** [comment]
I sadly cannot reproduce with that configuration either. If you have the ability to inspect the element and figure out where the background color is coming from it would be of great help.

**Comment by feerlessleadr** [comment]
Ok - I actually figured it out, and it was a stupid mistake on my part. I am using the ynab toolkit, and forgot that I had the setting where reconciled transactions have a dark grey text color and a green background (hence why I'm seeing that and you aren't). 

I have disabled this setting for now, and it works as it should. However, is it possible to allow the text color for reconciled transactions to be changed by ynab toolkit so that I can quickly differentiate between reconciled and non-reconciled transactions?
